packages/socket-mode/src/SlackWebSocket.spec.ts

```diff
@@ -1,14 +1,29 @@
 import { ConsoleLogger } from '@slack/logger';
 import { assert } from 'chai';
 import EventEmitter from 'eventemitter3';
+import proxyquire from 'proxyquire';
 import sinon from 'sinon';
-
-import { SlackWebSocket } from './SlackWebSocket';
+proxyquire.noPreserveCache();
```

Comment:
Superb test setups all throughout 👏  This adds so much confidence around the changes.

With some of the retry timing we might have to lookout for flakiness of tests due to the CI runners - nothing a rerun might not solve, but with the matrix setups I think our chances of this happening increase eversomuch 👾